features/data_tables.feature

```diff
@@ -59,29 +60,33 @@ Feature: Data tables
       | column_name |
       | Identifier  |
       | Title       |
+    And the table will include a column containing a link to the Folder detail page
 
   Scenario: All Digital Files table
     Given an instance running with some Digital Files created
     When a user visits the search page
     Then the page will use the Digital Files Elasticsearch index
     And the table will show the following columns and fields
-      | column_name   | es_field       |
-      | Filepath      | filepath       |
-      | Format        | fileformat     |
-      | Size (bytes)  | size_bytes     |
-      | Last modified | datemodified   |
-      | Folder        | dip.identifier | # link to Folder
+      | column_name   | es_field              |
+      | Filepath      | filepath              |
+      | Format        | fileformat            |
+      | Size (bytes)  | size_bytes            |
+      | Last modified | datemodified          |
+      | Folder        | dip.identifier        | # link to Folder
+      | Collection    | collection.identifier | # link to Collection
     And the search form will query over the following fields
       | es_field       |
       | filepath       |
       | fileformat     |
       | datemodified   |
       | dip.identifier |
     And the table header will show sort options for the following columns
-      | column_name  |
-      | Filepath     |
-      | Format       |
-      | Size (bytes) |
+      | column_name   |
+      | Filepath      |
+      | Format        |
+      | Size (bytes)  |
+      | Last modified |
```

Comment:
This field is currently being stored as a string field. We can do the same as we do with the identifier and title fields to allow sorting over it (indexing it two times in ES with different field types), but ideally it should be stored in the DB and the ES index as a date time field.

I've created #54 to take care of that in the future.